rfc9204.md

```diff
@@ -987,14 +988,14 @@ section and is referencing them, Required Insert Count will be greater than the
 Base, so the encoded difference is negative and the sign bit is set to 1.  If
 the field section was not encoded using representations that reference the most
 recent entry in the table and did not insert any new entries, the Base will be
-greater than the Required Insert Count, so the delta will be positive and the
-sign bit is set to 0.
+greater than the Required Insert Count, so the encoded difference will be
+positive and the sign bit is set to 0.
 
 The value of Base MUST NOT be negative. Though the protocol might operate
-correctly with a negative base using post-base indexing, it is unnecessary and
-inefficient. An endpoint MUST treat a field block with an S bit of 1 as invalid
-if the value of Required Insert Count is less than or equal to the value of
-Delta Base.
+correctly with a negative Base using post-Base indexing, it is unnecessary and
+inefficient. An endpoint MUST treat a field block with a sign bit of 1 as
```

Comment:
As a field, is it "sign" or "Sign"?

Reply:
"sign bit" is lower-case elsewhere, and is introduced with "one-bit sign ('S')" at the first reference in each section.  I'm open to changing it so that the field is called "Sign" and "S" is merely the representation in the diagram, but that's a (slightly) larger change.

Reply:
https://github.com/quicwg/base-drafts/pull/4989/commits/8f847d3f971e8e8a16396887fc5692d45b753957 shows what that change looks like, capitalizing Sign throughout.